package.json

```diff
@@ -67,13 +68,28 @@
     "request": "^2.88.2"
   },
   "scripts": {
-    "format-all": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\" *.js",
+    "format-all": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
     "build_and_serve": "yarn build; yarn serve",
-    "start": "craco start",
-    "build": "craco build",
+    "dev": "node --loader ts-node/esm src/server.ts",
```

Comment:
Er dette en ny kommando for å starte serveren lokalt? :)

Reply:
Jepp. Du kan selvfølgelig bare kjøre `serve` også, men denne hopper over kompileringssteget og kjører typescript direkte. Blir enklere å reloade når det skjer endringer da.